packages/uilib/src/lib/plugins/communication-explorer/sidebar/ied-accordion/ied-accordion.svelte

```diff
@@ -18,19 +19,42 @@
     $: relations = getConnectedIEDsByLabel(rootNode, IEDSelection.label)
     $: relationsByServiceType = groupRelationsByServiceType(relations)
     $: serviceTypes = Array.from(relationsByServiceType.entries())
+    $: details = getIEDDetails(rootNode, IEDSelection.label)
 
     const serviceTypeColor: { [key in MessageType | "Unknown"]: string } = {
     	GOOSE:         "--color-message-goose",
     	MMS:           "--color-message-mms",
     	SampledValues: "--color-message-sampledvalues",
     	Unknown:       "--color-message-unknown",
     }
+
+    let detailsCollapsed = true;
 </script>
 
 <div class="ied">
     <IED label={IEDSelection.label} isSelected={true} isSelectable={false} />
 </div>
-<div class="accordions">
+{#if details.length == 3}
+    <div class={detailsCollapsed ? 'details_collapsed' : ''}>
+    {#if details[0] != ""}
+        <h3>Logical Nodes</h3>
+        <p class="details">{@html details[0]}</p>
```

Comment:
`@html`should not be used if you don't sanitize your data, to prevent XSS attacks. We never know what is in the SCD files, we should not trust it.